lib/elixir/lib/module/locals_tracker.ex

```diff
@@ -196,6 +210,10 @@ defmodule Module.LocalsTracker do
   defp build_meta(nil), do: []
   defp build_meta(line), do: [line: line]
 
+  defp is_macro(:defmacro), do: true
+  defp is_macro(:defmacrop), do: true
```

Comment:
Maybe make it a `defguardp` as you have this logic here and also here `when kind in [:defmacro, :defmacrop]`? Or use a module attribute: `@macros [:defmacro, :defmacrop]` and do a `kind in @macros` in both.

I prefer the module attribute, but your calll.